### ISSUE TITLE
Data (content issues)

### ISSUE THREAD

**Issue by adibaba** [dice-group/dice-website#43]
Phone numbers should be set like:
+49 5251 60-3342
Currently, it is:
+49-525-16-03342

**Comment by yamalight** [comment]
Is that important? 🤔 
My phone could correctly recognize current format and suggested dialing it.

**Comment by adibaba** [comment]
Yes, especially in Germany it could be important. If some persons would see the current format, they could think the group can not handle common data formats. ;)

https://en.wikipedia.org/wiki/National_conventions_for_writing_telephone_numbers#Germany

**Comment by yamalight** [comment]
Makes sense. Will update 👍
